lenskit-integration-tests/src/it/gradle/train-test-all-output-files/build.gradle

```diff
@@ -61,6 +61,9 @@ task trainTest(type: TrainTest) {
         metric('ndcg') {
             columnName 'nDCG'
         }
+        metric('ndpm') {
+            columnName 'nDPM'
```

Comment:
We don't need a `columnName` here, just say `metric('ndpm')`. I don't even think we need to make NDPM's column name configurable — consider removing that logic entirely.